client/routes/index.js

```diff
@@ -33,7 +33,13 @@ export default [
             {
                 path: '/account/settings',
                 exact: true,
-                component: UserSettings
+                component: () => {
+                    return (
+                        <ConfirmAuth requiredPermission="tester">
```

Comment:
the access of an admin is everything the tester has access to + specially admin pages. Should we make that clearer on this page? And do something like `requiredPermission="any"` or something

Reply:
I think this seems fine and how one would normally reason about things.